google-cloud-datastore/src/test/java/com/google/cloud/datastore/ProtoTestData.java

```diff
@@ -63,6 +66,17 @@ public static PropertyReference propertyReference(String value) {
     return PropertyReference.newBuilder().setName(value).build();
   }
 
+  public static Aggregation countAggregation(String alias) {
+    return Aggregation.newBuilder().setAlias(alias).setCount(Count.newBuilder().build()).build();
+  }
+
+  public static Aggregation countAggregation(String alias, long limit) {
+    return Aggregation.newBuilder()
+        .setAlias(alias)
+        .setCount(Count.newBuilder().setUpTo(Int64Value.of(limit)))
```

Comment:
I don't think we want to use up to? Rather, can we use limit on the underlying query?